packages/fxa-auth-db-mysql/index.js

```diff
@@ -6,10 +6,12 @@ var config = require('./config');
 var dbServer = require('./db-server');
 var error = dbServer.errors;
 var logger = require('./lib/logging')('bin.server');
-var DB = require('./lib/db/mem')(logger, error);
+var DB = require('./lib/db/mysql')(logger, error);
 
 module.exports = function() {
   return DB.connect(config).then(function(db) {
-    return dbServer.createServer(db);
+    const server = dbServer.createServer(db);
```

Comment:
This was the key why mysql kept having too many connections, the database was not being closed when the test finished.